Rasa process starting
Added 'SpacyNLP' to component cache. Key 'SpacyNLP-en'.
Scheduler started
From /home/pvictor/miniconda3/envs/botenv/lib/python3.7/site-packages/tensorflow/python/ops/resource_variable_ops.py:435: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
From /home/pvictor/miniconda3/envs/botenv/lib/python3.7/site-packages/tensorflow/python/keras/backend.py:4010: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
From /home/pvictor/miniconda3/envs/botenv/lib/python3.7/site-packages/tensorflow/python/ops/math_ops.py:3066: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
Rasa Core server is up and running on http://localhost:5005
Failed to run custom action 'action_get_news'. Couldn't connect to the server at 'http://localhost:5055/webhook'. Is the server running? Error: HTTPConnectionPool(host='localhost', port=5055): Max retries exceeded with url: /webhook (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f7344054510>: Failed to establish a new connection: [Errno 111] Connection refused'))
Encountered an exception while running action 'action_get_news'. Bot will continue, but the actions events are lost. Make sure to fix the exception in your custom code.
Rasa process starting
Added 'SpacyNLP' to component cache. Key 'SpacyNLP-en'.
Scheduler started
From /home/pvictor/miniconda3/envs/botenv/lib/python3.7/site-packages/tensorflow/python/ops/resource_variable_ops.py:435: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
From /home/pvictor/miniconda3/envs/botenv/lib/python3.7/site-packages/tensorflow/python/keras/backend.py:4010: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
From /home/pvictor/miniconda3/envs/botenv/lib/python3.7/site-packages/tensorflow/python/ops/math_ops.py:3066: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
Rasa Core server is up and running on http://localhost:5005
Failed to run custom action 'action_get_news'. Couldn't connect to the server at 'http://localhost:5055/webhook'. Is the server running? Error: HTTPConnectionPool(host='localhost', port=5055): Max retries exceeded with url: /webhook (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fe4e4648490>: Failed to establish a new connection: [Errno 111] Connection refused'))
Encountered an exception while running action 'action_get_news'. Bot will continue, but the actions events are lost. Make sure to fix the exception in your custom code.
Rasa process starting
Added 'SpacyNLP' to component cache. Key 'SpacyNLP-en'.
Scheduler started
From /home/pvictor/miniconda3/envs/botenv/lib/python3.7/site-packages/tensorflow/python/ops/resource_variable_ops.py:435: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
From /home/pvictor/miniconda3/envs/botenv/lib/python3.7/site-packages/tensorflow/python/keras/backend.py:4010: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
From /home/pvictor/miniconda3/envs/botenv/lib/python3.7/site-packages/tensorflow/python/ops/math_ops.py:3066: to_int32 (from tensorflow.python.ops.math_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.cast instead.
Rasa Core server is up and running on http://localhost:5005
Failed to run custom action 'action_get_news'. Action server responded with a non 200 status code of 500. Make sure your action server properly runs actions and returns a 200 once the action is executed. Error: 500 Server Error: INTERNAL SERVER ERROR for url: http://localhost:5055/webhook
Encountered an exception while running action 'action_get_news'. Bot will continue, but the actions events are lost. Make sure to fix the exception in your custom code.
